engine/flink/components-api/src/main/scala/pl/touk/nussknacker/engine/flink/api/typeinformation/FlinkBaseTypeInfoRegister.scala

```diff
@@ -0,0 +1,62 @@
+package pl.touk.nussknacker.engine.flink.api.typeinformation
+
+import org.apache.flink.api.common.typeinfo.{TypeInfoFactory, TypeInformation, Types}
+import org.apache.flink.api.java.typeutils.TypeExtractor
+
+import java.lang.reflect.Type
+import java.time.{LocalDate, LocalDateTime, LocalTime}
+import java.util
+
+object FlinkBaseTypeInfoRegister {
```

Comment:
Please rename register -> registrar as we don't register anything in this register but the registrar does registration for us. BTW compare usages of these words in class names.